src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -534,6 +614,27 @@ public enum MidpointRounding
         AwayFromZero = 1,
         ToEven = 0,
     }
+    public sealed class OperatingSystem
+    {
+        private OperatingSystem() { }
+        public OperatingSystem(System.PlatformID platform, System.Version version) { }
+        public System.PlatformID Platform { get { return default(System.PlatformID); } }
+        public string ServicePack { get { return default(string); } }
+        public System.Version Version { get { return default(System.Version); } }
+        public object Clone() { return default(object); }
+        public override string ToString() { return default(string); }
+        public string VersionString { get { return default(string); } }
+    }
+    public enum PlatformID
+    {
+        Win32S = 0,
+        Win32Windows = 1,
+        Win32NT = 2,
+        WinCE = 3,
+        Unix = 4,
+        Xbox = 5,
+        MacOSX = 6
```

Comment:
This is the existing name.  We can't change it, as part of the goal here is binary compatibility.  In the future we could potentially add another name with the same value.